models/balancer/ethereum/balancer_ethereum_sources.yml

```diff
@@ -0,0 +1,50 @@
+version: 2
+
+sources: 
+  - name: balancer_ethereum
+    description: "Ethereum decoded tables related to Balancer contracts"
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
```

Comment:
a recent item tracked was not all decoded tables get new data within 24-hour time period. i didn't check myself yet, but wanted to call out here if you know off the top of your head that the data in the decoded table should truly be refreshed / loaded this often. the timeframe can be extended if needed